api/src/main/java/org/bf2/operator/Generated.java

```diff
@@ -0,0 +1,15 @@
+package org.bf2.operator;
```

Comment:
IMO this class should probably reside in common module in some util or code package

Reply:
We're already using Lombok for equals/hashcode implementations, I think I would rather use it for getters/setters than going with the annotation. The "generated" naming that is required for the annotation is a bit of a smell for code that isn't really generated.

Reply:
Oh, we are using Lombok? I couldn't find a Lombok.config so I thought this would be a better approach. @MikeEdgar  where should I look to find the lombok.config?

Reply:
We're using it, but the config is optional unless you want to do something like used `lombok.addLombokGeneratedAnnotation = true`. You can add the config along with your change.

Reply:
Yeah I was hoping to add that, so for further classes, the annotation gets added. Does that sound good?

Reply:
Yes, that's good. You can go ahead and remove any simple getter/setter methods in the `api` project and add the appropriate Lombok annotations to generate them as well.